Lib/ufo2ft/kernFeatureWriter.py

```diff
@@ -87,33 +87,41 @@ def write(self, linesep="\n"):
             self._splitRtlKerning()
 
         # write the lookups and feature
+        ltrKern = []
         if self.ltrScripts or not self.rtlScripts:
+            self._addKerning(ltrKern, self.glyphPairKerning)
+            self._addKerning(ltrKern, self.leftClassKerning, enum=True)
+            self._addKerning(ltrKern, self.rightClassKerning, enum=True)
+            self._addKerning(ltrKern, self.classPairKerning, ignoreZero=True)
+        if ltrKern:
             lines.append("lookup kern_ltr {")
-            self._addKerning(lines, self.glyphPairKerning)
-            self._addKerning(lines, self.leftClassKerning, enum=True)
-            self._addKerning(lines, self.rightClassKerning, enum=True)
-            self._addKerning(lines, self.classPairKerning, ignoreZero=True)
+            lines.extend(ltrKern)
             lines.append("} kern_ltr;")
             lines.append("")
 
+        rtlKern = []
         if self.rtlScripts:
-            lines.append("lookup kern_rtl {")
-            self._addKerning(lines, self.rtlGlyphPairKerning, rtl=True)
-            self._addKerning(lines, self.rtlLeftClassKerning, rtl=True,
+            self._addKerning(rtlKern, self.rtlGlyphPairKerning, rtl=True)
+            self._addKerning(rtlKern, self.rtlLeftClassKerning, rtl=True,
                              enum=True)
-            self._addKerning(lines, self.rtlRightClassKerning, rtl=True,
+            self._addKerning(rtlKern, self.rtlRightClassKerning, rtl=True,
                              enum=True)
-            self._addKerning(lines, self.rtlClassPairKerning, rtl=True,
+            self._addKerning(rtlKern, self.rtlClassPairKerning, rtl=True,
                              ignoreZero=True)
+        if rtlKern:
+            lines.append("lookup kern_rtl {")
+            lines.extend(rtlKern)
             lines.append("} kern_rtl;")
             lines.append("")
 
         lines.append("feature kern {")
-        if self.ltrScripts or not self.rtlScripts:
+        if (self.ltrScripts or not self.rtlScripts) and ltrKern:
```

Comment:
I was wondering about this. The font I’m working on has both RTL and LTR scripts, but kerning for the LTR script only. Right now my I end up with:
```
feature kern {
    lookup kern_ltr;
script latn;
language dflt;
lookup kern_ltr;
} kern;
```
Not sure why LTR kerning lookup is present twice, once with the global language systems and once with the LTR ones.